pipelines/DarkEnergyCamera/ApVerifyWithFakes.yaml

```diff
@@ -0,0 +1,30 @@
+# Gen 3 pipeline for ap_verify
+# This concatenates various lsst.verify metrics to an AP pipeline
+
+# This pipeline assumes $AP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml
+# has already been run.
+
+instrument: lsst.obs.decam.DarkEnergyCamera
+description: Fully instrumented AP pipeline with fakes, specialized for DECam
+imports:
+    - location: $AP_VERIFY_DIR/pipelines/ApVerifyWithFakes.yaml
+    # Do not use ap_pipe's ApPipe.yaml override, because ApVerifyWithFakes has
+    # *two* parallel AP pipelines.
```

Comment:
This comment is breaking my brain an little, what are the two parallel pipelines?

Reply:
I meant that there's processing done with and without fakes, and aside from `visitFakes` depending on non-fakes `calibrate` they're independent of each other (I think -- this pipeline breaks my brain, too).

Reply:
Ah OK. Maybe just update this comment to say, e.g., "The ap_pipe package has a standard ApPipe.yaml and a parallel one that includes fake/synthetic sources. We are intentionally not using the former here."